packages/react-devtools-inline/src/replay.js

```diff
@@ -0,0 +1,3 @@
+/** @flow */
+
+export {default as Store} from 'react-devtools-shared/src/devtools/store';
```

Comment:
Let's stick with named exports for this file. We may want to add other things?

Reply:
Good point. Pushed the changes.